esp/src/eclwatch/nls/hu/hpcc.js

```diff
@@ -428,7 +464,9 @@ define(
     SuperFile: "Super File",
     SuperFile: "Szuperfájl",
     Superfile: "Szuperfálj",
+    Superfiles: "Szuper-fájl",
     SuperFiles: "Szuperfájlok",
```

Comment:
I suspect they are different ("superfiles" versus "Super Files") - I should probably check the source, but probably won't in time for 5.2